bootstrap.sh

```diff
@@ -21,13 +21,18 @@ if [[ "${RELEASE}" == "master" || "${RELEASE}" == release-0.1 ]]; then
   set +x
   echo "NOTICE: the trace and pack commands are not available from master. RELEASE must be set to a released version (such as v0.2.0). See https://github.com/crossplane/crossplane-cli/releases for the full list of releases." >&2
   set -x
+  curl -sL -o "${PREFIX}"/bin/kubectl-crossplane https://raw.githubusercontent.com/crossplane/crossplane-cli/"${RELEASE}"/bin/kubectl-crossplane >/dev/null
+  curl -sL -o "${PREFIX}"/bin/kubectl-crossplane-krew-wrapper https://raw.githubusercontent.com/crossplane/crossplane-cli/"${RELEASE}"/bin/kubectl-crossplane-krew-wrapper >/dev/null
+  curl -sL -o "${PREFIX}"/bin/kubectl-crossplane-package https://raw.githubusercontent.com/crossplane/crossplane-cli/"${RELEASE}"/bin/kubectl-crossplane-package >/dev/null
   curl -sL -o "${PREFIX}"/bin/kubectl-crossplane-package-build https://raw.githubusercontent.com/crossplane/crossplane-cli/"${RELEASE}"/bin/kubectl-crossplane-package-build >/dev/null
   curl -sL -o "${PREFIX}"/bin/kubectl-crossplane-package-init https://raw.githubusercontent.com/crossplane/crossplane-cli/"${RELEASE}"/bin/kubectl-crossplane-package-init >/dev/null
   curl -sL -o "${PREFIX}"/bin/kubectl-crossplane-package-publish https://raw.githubusercontent.com/crossplane/crossplane-cli/"${RELEASE}"/bin/kubectl-crossplane-package-publish >/dev/null
   curl -sL -o "${PREFIX}"/bin/kubectl-crossplane-package-install https://raw.githubusercontent.com/crossplane/crossplane-cli/"${RELEASE}"/bin/kubectl-crossplane-package-install >/dev/null
   curl -sL -o "${PREFIX}"/bin/kubectl-crossplane-package-uninstall https://raw.githubusercontent.com/crossplane/crossplane-cli/"${RELEASE}"/bin/kubectl-crossplane-package-uninstall >/dev/null
   curl -sL -o "${PREFIX}"/bin/kubectl-crossplane-package-generate_install https://raw.githubusercontent.com/crossplane/crossplane-cli/"${RELEASE}"/bin/kubectl-crossplane-package-generate_install >/dev/null
   curl -sL -o "${PREFIX}"/bin/kubectl-crossplane-package-list https://raw.githubusercontent.com/crossplane/crossplane-cli/"${RELEASE}"/bin/kubectl-crossplane-package-list >/dev/null
+  curl -sL -o "${PREFIX}"/bin/kubectl-crossplane-registry https://raw.githubusercontent.com/crossplane/crossplane-cli/"${RELEASE}"/bin/kubectl-crossplane-package-registry >/dev/null
```

Comment:
these last 2 commands seem to add in `package` which isn't part of the source command name. e.g. `kubectl-crossplane-registry` => `kubectl-crossplane-package-registry`

Reply:
Oops sorry about that, need to stop and focus :sweat: